doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -725,6 +725,10 @@ Let's examine in detail the different elements included in the payload:
     way only to show the enormous flexibility of subscriptions.
 -   You can leave `conditions.attrs` empty to make a notification
     trigger on any entity attribute change (regardless of the name of the attribute).
+-   Notifications include the attribute values *after* processing the update operation
+    triggering the notification. However, you can make Orion to include also the
+    *previous* value. This is achieved using metadata. Have a look to
+    [the following piece of documentation](metadata.md#metadata-in-notifications).
```

Comment:
We don't use to include links to PRs in documentation. However, I understand that the topic is well covered by the materials in metadata.md cited from this point.

Thus, I'd say this is NTC.